### DISCUSSION TITLE
[Scam Alert] DARIK دریک

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#42] [Announcements]
<div dir="rtl">

**Project Name:** 
DARIK

**نام پروژه:** 
دریک

**نماد پروژه:**
DRC

**اتفاقی که افتاده است:**
نظرات صادر کننده توکن آقای باقر بزرگیان:

🔴دریک - با نماد DRC- Darik
سلام و عرض ارادت خدمت دوستان عزیز 
دوستان عزیز از این به بعد  در خصوص DRC یعنی دریک و شرکت  CompanyPoll  همه چیز شفاف و هیچ چیز مخفی و پنهانی نداریم 
در دست به ماشه مرحله اول و استارت  ۱۰۰ هزار  دلار  برای خرید دریک کاملا شفاف و عادلانه   روی تابلو  میزاریم 
یعنی  کسی که  DRCداشته باشه می تونه دریک را به ما بفروشه و دلار بگیره 
در چه قیمتی دریک را از شما می خریم ؟
قیمت  تاریخ  ساعت دقیقه و ثانیه را من  در کانال و لایو تعیین می کنم 
هرکس زودتر کلیک کنه  برنده است بنابراین داشتن دریک  برای فروش و سرعت کلیک کردن   حرف اول را در دست به ماشه  می زند 
دریک  را دوستان از کجا  دارن ؟
تعدادی در عرضه اولیه  فروختیم تعدادی هم در اف اکس کمپانی پول فروختیم و تعدادی هم در صرافی بیترا فروش رفته در صرافی بیترا کاملا شفاف  عرضه دریک ثابت است 
تعداد فروش رفته دریک از کجا معلوم است ؟
در صفحه  DRCو سایت   
👇🏻👇🏻👇🏻
 https://bscscan.com/token/0x611435c28097c03f264a605186e96d1eab71593d
کاملا مشخصه چه کیف پولی چقدر دریک داره و چه تعداد هولدر  است 
از کل یک میلیون دریک موجود 
۱۵۰ هزار دریک متعلق به  شرکت Companypoll است 
از این ۱۵۰ هزار دریک  ۵۰ هزار دریک  با قیمت ثابت  و عرضه ثابت  به ریال در صرافی بیترا گذاشتیم 
۵۰ هزار دریک  برای هزینه های شرکت  Companypollاست 
۵۰ هزار دریک هم برای  عملیات  دست به ماشه دست من است 
۸۰۰ هزار دریک برای وثیقه گذاری سهام وبرق   در کیف پول سخت افزاری نزد من است در لایو دست به ماشه کیف پول سخت افزاری را به دوستان نشان میدم
 ۵۰ هزار دریک هم برای کارمزد وثیقه گذاری در یک کیف پول جداگانه است 
کاملا شفاف و مشخصه یک دونه دریک از این کیف پولها جابجا بشه در بستر بلاک چین مشخص است 
چرا هولدرها را به ۵۰۰۰ نمی رسونن ؟
پیش خودشودشون فکر می کنن اگر هولدر بیارن تعداد زیاد میشه شانس خودشون برای برنده شدن و کلیک کردن کم میشه 
این افکار شیطانی است و با اصول ما سازگار نیست شما هولدر نیارید هولدر خارجی میارم  این صد هزار دلار و بعدا مبالغ بسیار بیشتر به هولدرهای خارجیان میرسه 
 من میخوام این پول به شما برسه 
دروغ نمیگم کار شیطان را انجام نمیدم افکار پلید و شیطانی ندارم مدیر عامل شرکت companypoll هستم  
مسیر الهی ما بن بست نداره  شکست نداره 
و این  مسیر  به شما  ثابت می شود 
پس هولدر بیارید و نگران این نباشید هولدر زیاد بشه  شانس برنده شدن شما کم می شود 
انشالله  پناه بر خدا 
Regards, 
Directing Manager Companypoll,
Bagher Bozorgian

طبق متنایی که توی کانال هست 
اسم ارز دریک با نماد DRC  و داخل سایت بایننس با پسوند org

https://explorer.binance.org/asset/DRC-FF4M

که هر دریک حدود مبلغ ۱۰۰ دلار  قیمتشه الان

و در سایت ایرانی  بیترا  هم قابل خرید هست

https://bitra.market/

🔴سپید نامه  رمز ارز  دریک(DRC)  با 
پشتوانه سهام وبرق
 دوستان عزیز سهامداران محترم وبرق در صورت تمایل و مطابق این سپیدنامه  می تونن از فردا جهت وثیقه گذاری و دریافت دریک به دفتر شرکت کارآفرین  ستاره بهارستان 
مراجعه کنند لطفا قبل از مراجعه به دفتر از طریق تلفنهای زیر   
تماس گرفته و هماهنگ کنید
👇🏻👇🏻👇🏻
 همراه(09374940956)
ثابت(02126156927)
آدرس و لوکیشن شرکت در  واتس آپ  همین شماره موبایل موجود است  متن قرارداد مطابق این سپیدنامه و سپید نامه پیوست قرارداد است 
ارادتمند محمد باقر  
ششم شهریور  ۱۴۰۰
[سپیدنامه.pdf](https://github.com/Webamoozcom/warning-list/files/7168155/default.pdf)

https://bscscan.com/token/0x611435c28097c03f264a605186e96d1eab71593d#balances
همونطور که می‌بینید ۸۰درصد این توکن برای یک نفر هستش؛ به این معنیه که این شخص هر موقع دلش بخواد میتونه شما رو دامپ کنه.

پ.ن:‌ ۱۰درصد هم برای یک والت دیگه هستش (که احتمالا اینم برای شخص کلاهبرداره)
از حادثه کریپتولند مردم باید درس بگیرن که اونجا هم همین سناریو پیاده شده بود و حدود ۹۲درصد توکنها مال یک والت بود و دیدیم که ۹۶درصد دامپ شد و سرمایه مردم به باد رفت.

🔴به نام  خداوند بخشنده مهربان 
مدیر عامل محترم شرکت سپرده گذاری مرکزی و تسویه وجوه 
با سلام و احترام 
شرکت Companypoll بستر  و زیر ساختهای لازم را برای معاملات  ۲۴ ساعته نماد وبرق به صورت دلاری (تتر) در سایت 
www.fxcompanypoll.org
فراهم کرده است 
چنانچه  بستر لازم  برای وثیقه گذاری و  آزادسازی وثیقه مطابق سپید نامه رمز ارز دریک لحظه ای و آنلاین باشد و همچنین معاملات  ۲۴ ساعته ریالی در شرکت فناوری اطلاعات بورس فراهم شود این شرکت آماده است بلافاصله معاملات دلاری  24ساعته نماد وبرق را آغاز کند 
جهت حذف کاغذ و بوروکراسی اداری همچنین شفافیت و دسترسی عادلانه به اطلاعات و سعی می شود  تا حد امکان مکاتبات  داخل ایران   از طریق همین کانال  انجام  شود  
Regards, 
Directing Manager Companypoll,
Bagher Bozorgian
https://t.me/c/1003491903/31067

**قوانینی که نقض شده است:**
بازفروش سهام بدون احراز هویت
قوانین ضد انحصار به دلیل نگهداری بیش از 80 درصد توکن نزد صادر کنندگان

@MohamadBagherBourse
https://t.me/joinchat/AAAAADvQEj_hmgakN1qHKQ

**نوع هشدار:**
هشدار ریسک بالا و عدم شفافیت

**نوع کلاهبرداری:** 
شید کوین

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**
باقر بزرگیان

**نام شرکت(ها):**
شرکت کار آفرین ستاره بهارستان
https://rasm.io/company/14008936134/%DA%A9%D8%A7%D8%B1%20%D8%A2%D9%81%D8%B1%DB%8C%D9%86%20%D8%B3%D8%AA%D8%A7%D8%B1%D9%87%20%D8%A8%D9%87%D8%A7%D8%B1%D8%B3%D8%AA%D8%A7%D9%86/

**آدرس وب سایت(ها):**
http://www.ecompanypoll.com/
http://www.fxcompanypoll.org/
https://www.bitdarik.com/
https://t.me/joinchat/AAAAADvQEj_hmgakN1qHKQ

 شماره تلفن و لوگوی شرکت و نام سهامدار شرکت در آگهی هایی که منتشر شده وجود دارد.

**برآورد مبلغ کلاهبرداری:**

**منابع:**
https://t.me/warning_list/3474

**جمع بندی:**

**Comment by saeedkousha** [comment]
***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from DAN YAR]
طبق متنایی که توی کانال هست 
اسم ارز دریک با نماد DRC  و داخل سایت بایننس با پسوند org

https://explorer.binance.org/asset/DRC-FF4M

که هر دریک حدود مبلغ ۱۰۰ دلار  قیمتشه الان

و در سایت ایرانی  بیترا  هم قابل خرید هست

چون اطلاعات زیادی نداشتم  گفتم بچه های گروه بررسیش کنن 

حتی وایت پیپیر هم داره این ارز براتون میفرستم

***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from کانال بورس پول (Bourse Poll) (محمد باقر بورس)]
[ File : سپیدنامه.pdf ]
🔴سپید نامه  رمز ارز  دریک(DRC)  با 
پشتوانه سهام وبرق
 دوستان عزیز سهامداران محترم وبرق در صورت تمایل و مطابق این سپیدنامه  می تونن از فردا جهت وثیقه گذاری و دریافت دریک به دفتر شرکت کارآفرین  ستاره بهارستان 
مراجعه کنند لطفا قبل از مراجعه به دفتر از طریق تلفنهای زیر   
تماس گرفته و هماهنگ کنید
👇🏻👇🏻👇🏻
 همراه(09374940956)
ثابت(02126156927)
آدرس و لوکیشن شرکت در  واتس آپ  همین شماره موبایل موجود است  متن قرارداد مطابق این سپیدنامه و سپید نامه پیوست قرارداد است 
ارادتمند محمد باقر  
ششم شهریور  ۱۴۰۰

***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from کانال بورس پول (Bourse Poll) (محمد باقر بورس)]
🔴دوستان لطفا توجه کنید  کیف پول تراست ولت مثل یک حساب بانکی  بین المللی است 
در هیچ کجای قانون هم ننوشته  که باز کردن کیف پول تراست  ولت  ممنوع است 
برای  شرکتها  برای اینکه شفاف باشه و حساب کتاب  برای حسابدار و حسابرس  مشخص بشه  صورتجلسه هیات مدیره  و آدرس کیف پول در آن قید  شود پسورد دست مدیر عامل
من نمی دونم حسابدار یا حسابرس  بانک مرکزی یا سازمان بورس گیر  میده یا نه خودتون مشکل خودتون رو با اینها حل کنید این مشکل شماست  
هرکسی هم میترسه   خدا نگهدار 
 در جزیره کیش   هرکس میخواد با ما کار کنه  تا زمانی که تحریم است  داشتن  کیف پول  ضرورت دارد و آدرس آن کیف پول به صورت سند محرمانه باید در شرکت  نگهداری شود 
حساب کتاب پول باید معلوم  باشد 
هر کاری دردسر خودشو داره 
پول درآوردن به همین راحتی نیست میخواید صادرات انجام بدید با شرایط تحریم کیف پول تراست ولت نداشته باشید؟
اگر پول درآوردن به همین راحتی بود خوب همه اینکار را می کردن 
👇🏻👇🏻👇🏻:
‏https://t.me/joinchat/AAAAADvQEj_hmgakN1qHKQ

***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from کانال بورس پول (Bourse Poll) (محمد باقر بورس)]
‎🔴آی بی - سئوالات دوستان 
درود
خسته نباشید جناب بزرگیان
لطفا اگه میشه توضیح بیش تری درباره آی بی بدین
و این که مثلا کسی که مثل من دانشجو هست چطور میتونه کار کنه
———————————-
🔴سلام ارادتمند 
اگر قبلا در کارگزاری پی سی ام ثبت نام و احراز هویت شدید  نیازی به احراز هویت مجدد نیست 
در غیر اینصورت  در سایت 
Www.fxcompanypoll.org
و ثبت نام وارد کابین تریدری شده و پس از تایید احراز هویت  تقاضای آی بی  بدهید 
بعد   از اینکه آی بی شما تایید شد یک لینک در اختیار شما قرار می گیرد 
در فضای مجازی تلگرام فیس بوک  اینستا و … می تونید لینک خود را معرفی کنید 
افرادی که از طریق لینک شما ثبت نام کنند هر معامله ای در بازارهای جهانی  بورس انرژی  بورس کالا انجام بدهند  در بازار نقدی  ۲۵ درصد  و در بازار غیر نقد هر لات  ۳ دلار به شما تعلق می گیرد 
اینجوری هم نیست که گنگ و مبهم باشد  شما در کابین تریدری لیست  مشتریان حجم معاملات و مقدار کارمزد را همواره می بینید و ماهانه به حساب شما واریز می شود 
اینکه چطوری کار کنید بستگی به    خودت داره 
یک نفر  شیاد اسم مانو جعل کرده بود از مردم دو نیم میلیون پول می گرفت ا پلیس فتا هم شکایت کردیم فایده نداشت 
الان  بیش از ۱۰۰ هزار ممبر داره  پول پارو می کند و همه مطالب منو به اسم خودش درج می کنه 
من نمیگم  مثل اون سرقت مطالب  و شیادی صورت گیرد 
مطمئناغ محکوم به شکست  است 
من میگم از این فرصت استفاده کنید 
شما با لینک آی بی خودت  و به اسم کمپانی پول هر تبلیغی خواستید انجام بدهید 
تحلیل بزار خبر بزار  و …. اگر انگلیسی بلدی به زبان انگلیسی  بنویس 
هنر آن است با دست خالی و بدون هیچ پولی  بتونی از راه صحیح پول  دربیاری 
 کار سخت و پیچیده ای نیست 
قبلا گفتم ما نوآنوری داشتیم  در فاز عملیاتی و اجرایی  وارد بازاریابی می شویم 
👇🏻👇🏻👇🏻:
‏https://t.me/joinchat/AAAAADvQEj_hmgakN1qHKQ

***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from کانال بورس پول (Bourse Poll) (محمد باقر بورس)]
🔴آی بی - پیامهای دوستان 
سلام استاد و وقت بخیر
میشه لطفا یه فایل راهنما در مورد درآمدزایی و اینکه چه کارهایی باید انجام بدیم از طریق CopmpanyPoll تو کانال بزارید تا یکم با جزییات بیشتری آشنا بشیم؟
———————————
🔴سلام ارادتمند 
بعد از اینکه تقاضای آی بی شما تایید شد 
یک لینک به شما اختصاص می یابد 
هر شخصی که از طریق لینک شما  ثبت نام کند  هر معامله ای انجام بدهد درصدی کارمزد به شما تعلق می گیرد
مثلا فرض کنید مشتری که با لینک شما ثبت نام کرده  به مبلغ ۱۰۰ هزار دلار خرما  با زعفران یا بنزین و …. معامله  می کند اگر ۲۰۰۰ دلار کارمزد این معامله باشد  ۶۰۰ دلار متعلق به شما است
 این بازار نقدی یا کش مارکتینگ است  در بازار غیر نقد فیوچرز و سی اف دی هم همینطور 
به صورت شفاف روزانه هفتگی و ماهانه لیست مشتریان معامله و کار مزد اختصاص یافته به خود را در کابین می بینید و سرماه  (میلادی ) دلار به حساب شما واریز می شود 
لینک آی بی خود را در  فضای مجازی  تلگرام فیس بوک  اینستا و …. برای داخلی و خارجی تبلیغ کنید بدیهی است هرچقدر مشتریان شما زیادتر شوند و معاملات بیشتری انجام بدهند  کارمزد بیشتری به شما تعلق می گیرد 
انگلیسی عربی ترکی کردی  و زبانهای دیگر  در هر کجا می تونید لینک خود را  تبلیغ کنید 
اخبار مطالب تحلیلی و غیره هم درج کنید اگر تحلیلگری بلد هستید می تونید تحلیل هم به مشتریان خود ارائه بدهید
نیاز به راهنمای خاصی نیست راهنمای تقاضای آی بی را در کانال گذاشتم  به محض اینکه مشتری شما  ثبت نام  کند همکاران ما در پشتیبانی  هر سئوالی داشته باشند پاسخ میدن  
   👇🏻👇🏻👇🏻:
‏https://t.me/joinchat/AAAAADvQEj_hmgakN1qHKQ

***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from کانال بورس پول (Bourse Poll) (محمد باقر بورس)]
🔴دادگاه علنی
جناب خواجه نصیر هنوز من به کیش نرفتم رفته از من شکایت کرده  مثلا خواستن پیش دستی کنن 
من هم به وکیل گفتم دادگاه باید علنی باشد 
می دونم از کجا آب  می خوره 
اینها  درج نماد  در بازارهای جهانی و آن ۵ درصد کارمزد را میخوان
حالا اسمشو بورس بین الملل کیش میزارن  یا هرچی  دست منه و این متعلق به  کودکان کار سهامداران کیان اعتماد و سهامداران کارآفرین است 
 آن مطلبی هم که سالهای گذشته در کانال نوشتم  به نقل از عصر ایران بوده 
اگر راست میگه بره از عصر ایران شکایت کنه 
از من شکایت می کنه  اونهم الان یادش افتاده شکایت کنه 
جناب خواجه نصیر  تندروها و اصولگراهای عزیز که الان همه قدرت در دست شماست ما را چی فرض کردید فکر کردید  کنترل و  کلید را دست شما میدم 
دادگاه علنی برگزار کنن تا همه کارهایی را که در بورس انجام دادن  برملا کنم 
در جزیره کیش  همه چیز شفاف است فکر کرده مامور ببره درخونه ما در کیش من می ترسم کیش نمیرم 
نه آقا از این خبرها نیست  
دوستان عزیز دریک  را بخرید با دست به ماشه ها که در کیش اجرا می کنم  تا قیمت بیت کوین دریک اسم داریوش بزرگ را  بالا می برم 
بستر بلاک چین است به کسی ربطی ندارد 
Regards, 
Directing Manager Companypoll,
Bagher Bozorgian

***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from کانال بورس پول (Bourse Poll) (محمد باقر بورس)]
🔴دریک - پیامهای دوستان 
سلام استاد عزیز در مورد دریک سوال داشتم چطوری امکان این هست دریک به بالاتر از بیت کوین برسه حجم نقدینگی زیادی باید به سمت دریک بیاد تا قیمتش به بیت کوین برسه
اگر امکانش هست توضیح بفرمایید با تشکر از شما
——————————————
🔴سلام ارادتمند 
کل دریک  ۱/۰۰۰/۰۰۰
است و همانطور که سپید نامه را  قبلا توضیح دادم 
از این یک میلیون ۱۵ درصد متعلق به مجری است 
پنج درصد کارمزد  وثیقه گذاری است  ۸۰درصد هم به وثیقه گذار  پرداخت  می شود 
تا ۵ سال فرصت وثیقه گذاری  است 
ممکنه  شرکت سپرده گذاری مرکزی و بورس وثیقه گذاری و آزاد سازی وثیقه را آنلاین و لحظه ای نکنند  
در این حالت  فقط آن ۱۵ درصد برای  معامله و خرید  و فروش وجود دارد 
چون از دریک برای ابزار تبادل  صادرات از بورس انرژی و صادرات از بورس کالا  استفاده می شود  چند  شرکت یا شخص حقیقی تبادل خارجی خود را با دریک انجام بدهند  قیمت دریک از بیت کوین هم بالاتر می رود 
صرفنظر از این موضوع  دریک اولین واحد پولی ایران و اسم داریوش بزرگ است 
  با دست به ماشه ها که در کیش  اجرا می کنم  قیمت دریک را  تا بیت کوین بالا می برم 
انرژی هم اولش فکر می کردن الکیه وقتی قیمتش به نزدیک ۵۰ هزار تومان تازه بعضی تازه فهمیدن گول خوردن  که تحت تاثیر جوسازی  کانالها و گروهها قرار گرفتن 
کتاب های دکتر احمد پور  پدر کارآفرینی ایران را بخونید  منبع قدرت در عصر حاضر دانایی است 
منبع قدرت زور بازو موشک و انرژی هسته ای نیست 
زیاد طول نمی کشه  خیلی زود متوجه می شوید  فرهنگ و تمدن ایرانی  چگونه در جزیره کیش  قدرت خود را به رخ دنیا می کشد 
۲۵۰۰ ساله دریک و اسم داریوش بزرگ  از بین نرفته و الان در موزه ها دریک است 
  قیمت دریک از بیت کوین هم پیشی می گیرد 
دریک با پشتوانه است  پشتوانه BNBهم بایننس است اما اغلب رمز ارزها  بدون پشتوانه  هستند 
Regards, 
Directing Manager Companypoll,
Bagher Bozorgian

***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from کانال بورس پول (Bourse Poll) (محمد باقر بورس)]
[ Photo ]
🔴من و همکارم آقای بستان ورود به جزیره کیش 
برای دریافت کارت کیشوندی یا باید شاغل باشید  یا ملک داشته باشید 
کودکان کار سهامداران کیان اعتماد تحلیل برخط  سهامداران  کارآفرین ستاره بهارستان همکار من هستند 
بنابراین توصیه می کنم در کابین تریدری  
www.fxcompanypoll.org
تقاضای آی بی بدهید که من بتونم  به عنوان همکار و شاغل در جزیره کیش شما را برای  دریافت کارت کیشوندی  معرفی کنم 
دقت کنیدمنظور  فقط  کودکان کار سهامداران کیان اعتماد و سهامداران کارآفرین ستاره بهارستان  است 
Regards, 
Directing Manager Companypoll,
Bagher Bozorgian

***, [۲۵.۰۹.۲۱ ۰۵:۵۶]
[Forwarded from DAN YAR]
خصوصا این یکی که  تراست ولت بسازن و ۱۲ کلمشو بدن دست کدیر عامل که خوده باقریان هست  این ۱۲ کلمه توس تراست ولت حکم  کلید گاو صندوق شخصی  است

**Comment by ImanKazemi** [comment]
اطلاعیه مهم 
بر اساس دستور مقام محترم قضایی، معاملات توکن «دریک» به علت تخلفات قانونی ناشر، تا اطلاع ثانوی مسدود می باشد.
(پلیس امنیت اقتصادی)
